<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NhapKhoWidget</class>
 <widget class="QWidget" name="NhapKhoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NhapKhoWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="congVanLabel">
     <property name="text">
      <string>Số công văn/lệnh nhập:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QCheckBox" name="moveDataCheckBox">
     <property name="text">
      <string>Di chuyển dữ liệu</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="congVanComboBox"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="themCongVanButton">
     <property name="text">
      <string>Thêm</string>
     </property>
     <property name="icon">
      <iconset resource="../MainWindow.qrc">
       <normaloff>:/icons/add.svg</normaloff>:/icons/add.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="7">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="dmsDirLabel">
       <property name="text">
        <string>Thư mục FTP:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dmsDirLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseDmsDirButton">
       <property name="text">
        <string>Chọn...</string>
       </property>
       <property name="icon">
        <iconset resource="../MainWindow.qrc">
         <normaloff>:/icons/folder.svg</normaloff>:/icons/folder.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="storeDirLabel">
       <property name="text">
        <string>Nơi lưu trữ:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="storeDirLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseStoreDirButton">
       <property name="text">
        <string>Chọn...</string>
       </property>
       <property name="icon">
        <iconset resource="../MainWindow.qrc">
         <normaloff>:/icons/folder.svg</normaloff>:/icons/folder.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="PhanLoaiTuLieuWidget" name="phanLoaiTuLieuWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="7">
    <widget class="ScanningWidget" name="scanningWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="7">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="progressPathLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="storeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Nhập kho</string>
       </property>
       <property name="icon">
        <iconset resource="../MainWindow.qrc">
         <normaloff>:/icons/vImporter.svg</normaloff>:/icons/vImporter.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="khoTuLieuLabel">
     <property name="text">
      <string>Tên kho:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="tenKhoLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PhanLoaiTuLieuWidget</class>
   <extends>QWidget</extends>
   <header>PhanLoaiTuLieuWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScanningWidget</class>
   <extends>QWidget</extends>
   <header>ScanningWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
